drivers/sensor/lm73c/doc/LM73CIMK-0.lbr

```diff
@@ -0,0 +1,138 @@
+<!--This work is licensed under a Creative Commons Attribution-ShareAlike 4.0 International License (CC BY-SA) with Design Exception 1.0-->
```

Comment:
We don't have sensor documentation like this in tree, and I don't think we should.  A link to the manufacturer's web product page in the sample's documentation, and in a comment in the driver, should be sufficient.